packages/ui/package.json

```diff
@@ -11,6 +11,9 @@
     "build-storybook": "storybook build"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material": "^5.15.12",
```

Comment:
Hm, are these required? These are kinda hefty packages that will require a lot of CPU time for just a JSON viewer.